cluster/migrations/000001_m3-initial.up.sql

```diff
@@ -1,5 +1,23 @@
 create schema provisioning;
 
+create table provisioning.admins
+(
+    id uuid
+        constraint admins_pk
+            primary key,
+    name varchar(256),
+    email varchar(256) not null,
+    access_key         varchar(256)                           not null,
+    sys_created_by varchar(256) not null,
+    sys_date_created timestamptz default now() not null
```

Comment:
all other tables have `sys_create_date`
```suggestion
    sys_created_date timestamptz default now() not null
```